src/java/org/apache/cassandra/db/Keyspace.java

```diff
@@ -770,12 +770,12 @@ public static Stream<Keyspace> allExisting()
 
     public static Iterable<Keyspace> nonSystem()
```

Comment:
This one seems to be use only in Keyspace.drain where we should use `nonLocalStrategy()` so we should be able to remove it.